addons/metadata.demo.tv/demo.py

```diff
@@ -46,156 +46,146 @@ def get_params():
-        liz.setProperty('video.first_aired', '2007-01-01')
-        liz.setProperty('video.thumbs', '2')
-        liz.setProperty('video.thumb1.url', 'DefaultBackFanart.png')
-        liz.setProperty('video.thumb1.aspect', '1.78')
-        liz.setProperty('video.thumb2.url', '/home/akva/Pictures/hawaii-shirt.png')
-        liz.setProperty('video.thumb2.aspect', '2.35')
-        liz.setProperty('video.genre','Action / Comedy')
-        liz.setProperty('video.country', 'Norway / Sweden / China')
-        liz.setProperty('video.writing_credits', 'None / Want / To Admit It')
-        liz.setProperty('video.director', 'spiff / spiff2')
-        liz.setProperty('video.actors', '2')
-        liz.setProperty('video.actor1.name', 'spiff')
-        liz.setProperty('video.actor1.role', 'himself')
-        liz.setProperty('video.actor1.sort_order', '2')
-        liz.setProperty('video.actor1.thumb', '/home/akva/Pictures/fish.jpg')
-        liz.setProperty('video.actor1.thumb_aspect', 'poster')
-        liz.setProperty('video.actor2.name', 'monkey')
-        liz.setProperty('video.actor2.role', 'orange')
-        liz.setProperty('video.actor2.sort_order', '1')
-        liz.setProperty('video.actor1.thumb_aspect', '1.78')
-        liz.setProperty('video.actor2.thumb', '/home/akva/Pictures/coffee.jpg')
-        liz.setProperty('video.date_added', '2016-01-01')
+        liz.setInfo('video',
+                    {'title': 'Demo Episode 1',
+                     'originaltitle': 'Demo æpisod 1x1',
+                     'sorttitle': '2',
+                     'episode': 1,
+                     'season': 1,
+                     'userrating': 5,
+                     'plotoutline': 'Outline yo',
+                     'plot': 'Plot yo',
+                     'tagline': 'Tag yo',
+                     'duration': 110,
+                     'mpaa': 'T',
+                     'trailer': '/home/akva//porn/bukkake.mkv',
+                     'genre': ['Action', 'Comedy'],
+                     'country': ['Norway', 'Sweden', 'China'],
+                     'credits': ['None', 'Want', 'To Admit It'],
+                     'director': ['spiff', 'spiff2'],
+                     'studio': ['Studio1', 'Studio2'],
+                     'dateadded': '2016-01-01',
+                     'premiered': '2015-01-01',
+                     'aired': '2007-01-01',
+                     'tag': ['Porn', 'Umomz']
+                    })
+        liz.setRating("imdb", 9, 100000, True )
+        liz.setRating("tvdb", 8.9, 1000)
+        liz.setUniqueIDs({ 'tvdb': '3894', 'imdb' : 'tt384940' }, 'tvdb')
+        liz.addSeason(1, 'Horrible')
+        liz.addSeason(2, 'Crap')
+        liz.setCast([{'name': 'spiff', 'role': 'himself', 'thumbnail': '/home/akva/Pictures/fish.jpg', 'order': 2},
+                    {'name': 'monkey', 'role': 'orange', 'thumbnail': '/home/akva/Pictures/coffee.jpg', 'order': 1}])
+        liz.addAvailableThumb('DefaultBackFanart.png', '1.78')
```

Comment:
Like the TV show itself above, the second parameter to `addAvailableThumb` should be an art type, generally 'thumb', or could even something wild like 'clearlogo'.

Reply:
The aspect in the line above and below still need adjusted, but otherwise this PR looks good.